src/libraries/System.Reflection.MetadataLoadContext/src/System/ThrowHelper.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// This file defines an internal static class used to throw exceptions in the
+// the System.Reflection.MetadataLoadContext code.
+
+using System.Reflection;
+
+namespace System
+{
+    internal static class ThrowHelper
+    {
+        internal static AmbiguousMatchException GetAmbiguousMatchException(MemberInfo memberInfo)
```

Comment:
Kinda hating that we have the same helpers and strings in multiple places, but I don't know/think this is worth a public API change or making internals visible to others...